airflow/providers/exasol/hooks/exasol.py

```diff
@@ -108,6 +108,37 @@ def get_first(self, sql: Union[str, list], parameters: Optional[dict] = None) ->
             with closing(conn.execute(sql, parameters)) as cur:
                 return cur.fetchone()
 
+    def export_to_file(
```

Comment:
does it offer the choice of file format? (json, csv, other?)
The method name doesn't suggest it's generic so suitable for more than one file extension but there is no file extension parameter to config

Reply:
I think what @turbaszek has nicely done here is to allow Exasol users to seamlessly call what `pyexasol` supports. Please refer to https://github.com/badoo/pyexasol/blob/master/pyexasol/connection.py#L259 

I don't know Exasol well, but based on the implementation here, this new method supports whatever the original method in `Pyexasol` supports. So to my understanding, the question you raised should not be a concern.

Let me know if this makes sense to you?

Reply:
@XD-DENG is right - I hoped that using the `Pyexasol` method will give us most flexibility and will be easiest to maintain. Here is the exasol info:


>The target file can either be CSV or FBV files and should comply to the format specifications in the CSV Data Format and the Fixblock Data format (FBV). File names may only consist of ASCII characters. A BOMClosed is not supported.

https://docs.exasol.com/sql/export.htm#UsageNotes